vnext/Microsoft.ReactNative/CoreAppPage.cpp

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#include "pch.h"
+#include "CoreAppPage.h"
+#include "ReactApplication.h"
+
+#if __has_include("CoreAppPage.g.cpp")
+#include "CoreAppPage.g.cpp"
+#endif
+
+using namespace winrt;
+using namespace Windows::UI::Xaml;
+
+namespace winrt::Microsoft::ReactNative::implementation {
+CoreAppPage::CoreAppPage() {
+  InitializeComponent();
+  auto app = xaml::Application::Current().as<winrt::Microsoft::ReactNative::ReactApplication>();
+  auto impl_app = winrt::get_self<winrt::Microsoft::ReactNative::implementation::ReactApplication>(app);
```

Comment:
>impl_app [](http://example.com/codeflow?start=7&length=8)

nit: it is better that we follow the same style to name variables in our project. In the rest of the code we use camelCase for local variables and parameters. E.g. implApp and pageNavigated.